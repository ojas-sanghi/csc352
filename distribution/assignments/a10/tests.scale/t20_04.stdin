71932532196941215756
72326688015145201171
64247746609481285845
71212098368004491230
18754617351408903566
82102377218800308811
13135803473418606358
81767501154728239579
39644131691874859879
25799627675649330963
23313111167957810682
83615197775318209963
1516943653546082863
58283761263564989542
9491000510189494006
68915982189171136870
12139563819865597739
79519141952166956987
76070602392400922228
38162288129759320100
