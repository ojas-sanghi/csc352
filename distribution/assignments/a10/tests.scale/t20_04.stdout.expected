71932532196941215756 (audit=1)
scaled by 1: 71932532196941215756 (audit=1)
scaled by 2: 143865064393882431512 (audit=1)
scaled by 3: 431595193181647294536 (audit=1)
scaled by 4: 1726380772726589178144 (audit=1)
scaled by 5: 8631903863632945890720 (audit=1)
scaled by 6: 51791423181797675344320 (audit=1)
scaled by 7: 362539962272583727410240 (audit=1)
scaled by 8: 2900319698180669819281920 (audit=1)
scaled by 9: 26102877283626028373537280 (audit=1)
72326688015145201171 (audit=1)
scaled by 1: 72326688015145201171 (audit=1)
scaled by 2: 144653376030290402342 (audit=1)
scaled by 3: 433960128090871207026 (audit=1)
scaled by 4: 1735840512363484828104 (audit=1)
scaled by 5: 8679202561817424140520 (audit=1)
scaled by 6: 52075215370904544843120 (audit=1)
scaled by 7: 364526507596331813901840 (audit=1)
scaled by 8: 2916212060770654511214720 (audit=1)
scaled by 9: 26245908546935890600932480 (audit=1)
64247746609481285845 (audit=1)
scaled by 1: 64247746609481285845 (audit=1)
scaled by 2: 128495493218962571690 (audit=1)
scaled by 3: 385486479656887715070 (audit=1)
scaled by 4: 1541945918627550860280 (audit=1)
scaled by 5: 7709729593137754301400 (audit=1)
scaled by 6: 46258377558826525808400 (audit=1)
scaled by 7: 323808642911785680658800 (audit=1)
scaled by 8: 2590469143294285445270400 (audit=1)
scaled by 9: 23314222289648569007433600 (audit=1)
71212098368004491230 (audit=1)
scaled by 1: 71212098368004491230 (audit=1)
scaled by 2: 142424196736008982460 (audit=1)
scaled by 3: 427272590208026947380 (audit=1)
scaled by 4: 1709090360832107789520 (audit=1)
scaled by 5: 8545451804160538947600 (audit=1)
scaled by 6: 51272710824963233685600 (audit=1)
scaled by 7: 358908975774742635799200 (audit=1)
scaled by 8: 2871271806197941086393600 (audit=1)
scaled by 9: 25841446255781469777542400 (audit=1)
18754617351408903566 (audit=1)
scaled by 1: 18754617351408903566 (audit=1)
scaled by 2: 37509234702817807132 (audit=1)
scaled by 3: 112527704108453421396 (audit=1)
scaled by 4: 450110816433813685584 (audit=1)
scaled by 5: 2250554082169068427920 (audit=1)
scaled by 6: 13503324493014410567520 (audit=1)
scaled by 7: 94523271451100873972640 (audit=1)
scaled by 8: 756186171608806991781120 (audit=1)
scaled by 9: 6805675544479262926030080 (audit=1)
82102377218800308811 (audit=1)
scaled by 1: 82102377218800308811 (audit=1)
scaled by 2: 164204754437600617622 (audit=1)
scaled by 3: 492614263312801852866 (audit=1)
scaled by 4: 1970457053251207411464 (audit=1)
scaled by 5: 9852285266256037057320 (audit=1)
scaled by 6: 59113711597536222343920 (audit=1)
scaled by 7: 413795981182753556407440 (audit=1)
scaled by 8: 3310367849462028451259520 (audit=1)
scaled by 9: 29793310645158256061335680 (audit=1)
13135803473418606358 (audit=1)
scaled by 1: 13135803473418606358 (audit=1)
scaled by 2: 26271606946837212716 (audit=1)
scaled by 3: 78814820840511638148 (audit=1)
scaled by 4: 315259283362046552592 (audit=1)
scaled by 5: 1576296416810232762960 (audit=1)
scaled by 6: 9457778500861396577760 (audit=1)
scaled by 7: 66204449506029776044320 (audit=1)
scaled by 8: 529635596048238208354560 (audit=1)
scaled by 9: 4766720364434143875191040 (audit=1)
81767501154728239579 (audit=1)
scaled by 1: 81767501154728239579 (audit=1)
scaled by 2: 163535002309456479158 (audit=1)
scaled by 3: 490605006928369437474 (audit=1)
scaled by 4: 1962420027713477749896 (audit=1)
scaled by 5: 9812100138567388749480 (audit=1)
scaled by 6: 58872600831404332496880 (audit=1)
scaled by 7: 412108205819830327478160 (audit=1)
scaled by 8: 3296865646558642619825280 (audit=1)
scaled by 9: 29671790819027783578427520 (audit=1)
39644131691874859879 (audit=1)
scaled by 1: 39644131691874859879 (audit=1)
scaled by 2: 79288263383749719758 (audit=1)
scaled by 3: 237864790151249159274 (audit=1)
scaled by 4: 951459160604996637096 (audit=1)
scaled by 5: 4757295803024983185480 (audit=1)
scaled by 6: 28543774818149899112880 (audit=1)
scaled by 7: 199806423727049293790160 (audit=1)
scaled by 8: 1598451389816394350321280 (audit=1)
scaled by 9: 14386062508347549152891520 (audit=1)
25799627675649330963 (audit=1)
scaled by 1: 25799627675649330963 (audit=1)
scaled by 2: 51599255351298661926 (audit=1)
scaled by 3: 154797766053895985778 (audit=1)
scaled by 4: 619191064215583943112 (audit=1)
scaled by 5: 3095955321077919715560 (audit=1)
scaled by 6: 18575731926467518293360 (audit=1)
scaled by 7: 130030123485272628053520 (audit=1)
scaled by 8: 1040240987882181024428160 (audit=1)
scaled by 9: 9362168890939629219853440 (audit=1)
23313111167957810682 (audit=1)
scaled by 1: 23313111167957810682 (audit=1)
scaled by 2: 46626222335915621364 (audit=1)
scaled by 3: 139878667007746864092 (audit=1)
scaled by 4: 559514668030987456368 (audit=1)
scaled by 5: 2797573340154937281840 (audit=1)
scaled by 6: 16785440040929623691040 (audit=1)
scaled by 7: 117498080286507365837280 (audit=1)
scaled by 8: 939984642292058926698240 (audit=1)
scaled by 9: 8459861780628530340284160 (audit=1)
83615197775318209963 (audit=1)
scaled by 1: 83615197775318209963 (audit=1)
scaled by 2: 167230395550636419926 (audit=1)
scaled by 3: 501691186651909259778 (audit=1)
scaled by 4: 2006764746607637039112 (audit=1)
scaled by 5: 10033823733038185195560 (audit=1)
scaled by 6: 60202942398229111173360 (audit=1)
scaled by 7: 421420596787603778213520 (audit=1)
scaled by 8: 3371364774300830225708160 (audit=1)
scaled by 9: 30342282968707472031373440 (audit=1)
1516943653546082863 (audit=1)
scaled by 1: 1516943653546082863 (audit=1)
scaled by 2: 3033887307092165726 (audit=1)
scaled by 3: 9101661921276497178 (audit=1)
scaled by 4: 36406647685105988712 (audit=1)
scaled by 5: 182033238425529943560 (audit=1)
scaled by 6: 1092199430553179661360 (audit=1)
scaled by 7: 7645396013872257629520 (audit=1)
scaled by 8: 61163168110978061036160 (audit=1)
scaled by 9: 550468512998802549325440 (audit=1)
58283761263564989542 (audit=1)
scaled by 1: 58283761263564989542 (audit=1)
scaled by 2: 116567522527129979084 (audit=1)
scaled by 3: 349702567581389937252 (audit=1)
scaled by 4: 1398810270325559749008 (audit=1)
scaled by 5: 6994051351627798745040 (audit=1)
scaled by 6: 41964308109766792470240 (audit=1)
scaled by 7: 293750156768367547291680 (audit=1)
scaled by 8: 2350001254146940378333440 (audit=1)
scaled by 9: 21150011287322463405000960 (audit=1)
9491000510189494006 (audit=1)
scaled by 1: 9491000510189494006 (audit=1)
scaled by 2: 18982001020378988012 (audit=1)
scaled by 3: 56946003061136964036 (audit=1)
scaled by 4: 227784012244547856144 (audit=1)
scaled by 5: 1138920061222739280720 (audit=1)
scaled by 6: 6833520367336435684320 (audit=1)
scaled by 7: 47834642571355049790240 (audit=1)
scaled by 8: 382677140570840398321920 (audit=1)
scaled by 9: 3444094265137563584897280 (audit=1)
68915982189171136870 (audit=1)
scaled by 1: 68915982189171136870 (audit=1)
scaled by 2: 137831964378342273740 (audit=1)
scaled by 3: 413495893135026821220 (audit=1)
scaled by 4: 1653983572540107284880 (audit=1)
scaled by 5: 8269917862700536424400 (audit=1)
scaled by 6: 49619507176203218546400 (audit=1)
scaled by 7: 347336550233422529824800 (audit=1)
scaled by 8: 2778692401867380238598400 (audit=1)
scaled by 9: 25008231616806422147385600 (audit=1)
12139563819865597739 (audit=1)
scaled by 1: 12139563819865597739 (audit=1)
scaled by 2: 24279127639731195478 (audit=1)
scaled by 3: 72837382919193586434 (audit=1)
scaled by 4: 291349531676774345736 (audit=1)
scaled by 5: 1456747658383871728680 (audit=1)
scaled by 6: 8740485950303230372080 (audit=1)
scaled by 7: 61183401652122612604560 (audit=1)
scaled by 8: 489467213216980900836480 (audit=1)
scaled by 9: 4405204918952828107528320 (audit=1)
79519141952166956987 (audit=1)
scaled by 1: 79519141952166956987 (audit=1)
scaled by 2: 159038283904333913974 (audit=1)
scaled by 3: 477114851713001741922 (audit=1)
scaled by 4: 1908459406852006967688 (audit=1)
scaled by 5: 9542297034260034838440 (audit=1)
scaled by 6: 57253782205560209030640 (audit=1)
scaled by 7: 400776475438921463214480 (audit=1)
scaled by 8: 3206211803511371705715840 (audit=1)
scaled by 9: 28855906231602345351442560 (audit=1)
76070602392400922228 (audit=1)
scaled by 1: 76070602392400922228 (audit=1)
scaled by 2: 152141204784801844456 (audit=1)
scaled by 3: 456423614354405533368 (audit=1)
scaled by 4: 1825694457417622133472 (audit=1)
scaled by 5: 9128472287088110667360 (audit=1)
scaled by 6: 54770833722528664004160 (audit=1)
scaled by 7: 383395836057700648029120 (audit=1)
scaled by 8: 3067166688461605184232960 (audit=1)
scaled by 9: 27604500196154446658096640 (audit=1)
38162288129759320100 (audit=1)
scaled by 1: 38162288129759320100 (audit=1)
scaled by 2: 76324576259518640200 (audit=1)
scaled by 3: 228973728778555920600 (audit=1)
scaled by 4: 915894915114223682400 (audit=1)
scaled by 5: 4579474575571118412000 (audit=1)
scaled by 6: 27476847453426710472000 (audit=1)
scaled by 7: 192337932173986973304000 (audit=1)
scaled by 8: 1538703457391895786432000 (audit=1)
scaled by 9: 13848331116527062077888000 (audit=1)
